src/Application/Gallery/Gallery.tsx

```diff
@@ -63,9 +62,8 @@ class Gallery extends React.Component<Props, State> {
     }
   }
 
-  componentDidUpdate(prevProps: Props, prevState: State) {
+  componentDidUpdate() {
     if (
-      !prevState.visible &&
```

Comment:
why need to remove this line?

Reply:
I think maybe we should add this to `componentDidMount`
```
if (this.state.visible) {
      this.sliderRef.current.sliderContainerRef.current.focus();
}
```

Reply:
Added this condition to componentDidMount.
But I think if this is added, we don't need the previous condition. As whenever the modal is made visible, the focus is added to the slider with this condition.
But maybe can remove the previous condition after writing tests for this component?

Reply:
`componentDidMount` only being executed once, so I think we still need the condition in `componentDidUpdate` right?

Yes adding test is needed, please add thank you!

Reply:
Yes yes, sorry wrote the reply wrong, I have added it to componentDidUpdate only!

Reply:
Added to `componentDidMount` and opened an issue https://github.com/glints-dev/glints-aries/issues/374